app/controllers/dashboards_controller.rb

```diff
@@ -22,9 +31,29 @@ def export
     redirect_to dashboards_path, alert: e.message
   end
 
+  def push
+    version = Version.preview_version
+
+    if version.blank?
+      flash.alert = 'No preview version available'
+    else
+      pv = Version.create(number: version.number, production: true, user: current_user)
+
+      if pv.persisted?
+        flash.notice = 'Production data updated'
+      else
+
+        flash.alert = "Production data not updated: #{version.inspect} #{pv.errors.inspect}"
+      end
+    end
+
+    redirect_to dashboards_path
```

Comment:
Actually, that was intentional. Push has no parameters, it merely makes the latest preview version into a production version. The versioning is really abstracted away from the user, and I wanted the push to be simply a pass fail thing for them